ocs_ci/helpers/helpers.py

```
@@ -1790,38 +1789,67 @@ def get_provision_time(interface, pvc_name, status="start"):
-    logs = pod.get_pod_logs(pod_name[0], "csi-provisioner")
-    logs += pod.get_pod_logs(pod_name[1], "csi-provisioner")
 
-    logs = logs.split("\n")
-    # Extract the time for the one PVC provisioning
+    # Get logs from both provisioner pods
+    logs1 = pod.get_pod_logs(pod_name[0], "csi-provisioner").split("\n")
+    logs2 = pod.get_pod_logs(pod_name[1], "csi-provisioner").split("\n")
+    logs = logs1 + logs2
+    logger.info(f"total logs loaded: {len(logs)}")
+
+    def find_events(name):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `find_events`

(ANN202)